internal/planner/select.go

```diff
@@ -61,6 +61,8 @@ type selectTopNode struct {
 	// selectNode is used pre-wiring of the plan (before expansion and all).
 	selectNode *selectNode
 
+	similarity []*similarityNode
```

Comment:
todo: Please add a comment (preferably linked to a follow-up ticket), noting what this is, and immediate future plans.